source/macos/AdaptiveCards/AdaptiveCards/Views/ACRCollectionView.swift

```diff
@@ -59,32 +66,49 @@ class ACRCollectionView: NSScrollView {
         autohidesScrollers = true
     }
     
+    public func containImageViewCell(with id: String) -> NSView? {
+        if let imgView = imageViews.first(where: { $0.identifier?.rawValue == id }) {
+            return imgView
+        }
+        return nil
+    }
+    
+    private func reloadData() {
+        invalidateIntrinsicContentSize()
+        collectionView.reloadData()
+    }
+    
+    override func invalidateIntrinsicContentSize() {
+        pContentSize = nil
+        super.invalidateIntrinsicContentSize()
+    }
```

Comment:
Nit: override function in middle of private functions, can we group them together